static/js/components/EducationTab.js

```diff
@@ -105,7 +105,7 @@ class EducationTab extends ProfileTab {
     const {profile} = this.props;
     return profile['education'].map(education => {
       if (education.degree_name === level.value && 'id' in education) {
-        return <Grid className="existing-education-grid">
+        return <Grid key={"education-row-" + education.id} className="existing-education-grid">
```

Comment:
For future reference you only need `education.id` here since keys only need to be unique compared to other keys in the same collection. It's not like the `id` attribute which needs to be unique across all elements in the browser